titus-server-master/src/main/java/com/netflix/titus/master/kubernetes/pod/taint/DefaultTaintTolerationFactory.java

```diff
@@ -49,13 +49,12 @@ public DefaultTaintTolerationFactory(KubePodConfiguration configuration,
     }
 
     @Override
-    public List<V1Toleration> buildV1Toleration(Job<?> job, Task task, boolean useKubeScheduler) {
+    public List<V1Toleration> buildV1Toleration(Job<?> job, Task task) {
         List<V1Toleration> tolerations = new ArrayList<>();
 
         // Default taints.
         tolerations.add(Tolerations.TOLERATION_VIRTUAL_KUBLET);
-        V1Toleration schedulerToleration = useKubeScheduler ? Tolerations.TOLERATION_KUBE_SCHEDULER : Tolerations.TOLERATION_FENZO_SCHEDULER;
-        tolerations.add(schedulerToleration);
+        tolerations.add(Tolerations.TOLERATION_KUBE_SCHEDULER);
```

Comment:
What do you think about eventually removing this taint from all agents, so that we can remove this toleration eventually, cleaning up our pod specs?

Reply:
I think it makes sense to remove it. But we should start with agents first.